judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/problem/bundle/ItemConfig.java

```diff
@@ -1,5 +1,21 @@
 package judgels.sandalphon.api.problem.bundle;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.function.Function;
+
 public interface ItemConfig {
     String getStatement();
+    ItemConfig processDisplayText(Function<String, String> processor);
```

Comment:
The more canonical name is:

```suggestion
    ItemConfig transformContentText(Function<String, String> transformer);
```

(display -> content to be specific, because for example "alias" is also a displayed text but is not actually transformed)

Reply:
Renamed